### PR TITLE
published to npm

### PR DESCRIPTION
**Pull request by keroserene**
Minor tweaks to prepare as npm module so it can be included in uProxy.

https://www.npmjs.org/package/socks-rtc

Tested: End-to-end

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,25 +9,45 @@ This is built on top of [freedom](https://github.com/UWNetworksLab/freedom).
...
+- If you want to run the example Chrome App, run `grunt chrome`.
+
+#### Usage
+
+To make use of this library, one needs to include `socks-to-rtc.json` and
+`rtc-to-net.json`, which are manifests describing freedom modules, as
+dependencies in the parent application's freedom manifest. Three things must occur
+for the two components to speak to each other:
+
+- `.emit('start')` to _rtc-to-net_ begins the remote peer server.
```
Comment (iislucas) [inline README.md]: Can you clarify the object to call this on? / where this happens?

File: README.md [hunk 2]
```diff
@@ -9,25 +9,45 @@ This is built on top of [freedom](https://github.com/UWNetworksLab/freedom).
...
+- `.emit('start', { host, port, peerId })` to _socks-to-rtc_ begins listening
+  locally.
+- Establish a signalling channel between _rtc-to-net_ and _socks-to-rtc_ so that
```
Comment (iislucas) [inline README.md]: Can you say a tiny bit about how this happens, e.g. done by start calls, or some other call?

File: README.md [hunk 3]
```diff
@@ -41,9 +61,15 @@ running locally (on localhost:4444). To do this:
  - run the test with `grunt endtoend`
```
Comment (iislucas) [inline README.md:64]: Does this run Trevor's tests? can we do this too? :)

File: README.md [hunk 4]
```diff
@@ -41,9 +61,15 @@ running locally (on localhost:4444). To do this:
...
+  unpacked extension the `socks-rtc/chrome/` directory.
+- Open the background page, which will start a socks-rtc proxy listening on
+  localhost:9999.
+
 At the moment, the way to test that this works is to just curl a webpage
-through the proxy which socks-rtc sets up if you've built it successfully.
-For example:
+through the socks-rtc proxy. For example:
 
 `curl -x socks5h://localhost:9999 www.google.com`
```
Comment (iislucas) [inline README.md:78]: Worth mentioning that the -x socks5h means that DNS requests are make by the proxy's networking stack; DNS is not resolved locally.

**Comment by iislucas** [comment]
LGTM. Awesome! just a few comment suggestions.

File: README.md (hunk 3 above)
Comment (keroserene) [inline README.md:64]: I'm assuming it does, as I think Trevor put this line here :)
Perhaps it's possible to get selenium to do the start-stop tests

**Pull request merged**